browser-tests/utils/settings.ts

```diff
@@ -1,24 +1,28 @@
 import * as dotenv from 'dotenv';
 
+// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+const get = require('lodash/get');
```

Comment:
The typescript setting Testcafe uses do not allow default imports and I didn't spend too much time debugging.